container/go/pkg/oci/reader_test.go

```diff
@@ -0,0 +1,123 @@
+package oci
+
+import (
+	"testing"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/types"
+	"github.com/google/go-containerregistry/pkg/v1/validate"
+)
+
+var readertests = []struct {
```

Comment:
Comment here explaining what each test case in readerTest does.

`// readerTests has test cases for the Read function that loads a docker image stored on disk serialized in OCI format by "Write"`